support-frontend/assets/helpers/subscriptionsForms/formReducer.ts

```diff
@@ -32,15 +32,16 @@ function createFormReducer(
 
 	const { productPrices, orderIsAGift } = window.guardian;
 	const initialState = {
-		stage: 'checkout',
+		stage: 'checkout' as Stage,
```

Comment:
not sure why the cast is necessary here?

Reply:
maybe you'd need to explicitly type `initialState`?

Reply:
🤦  of course!